src/components/flatSelect/tests/FlatSelectConnected.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import { TestUtils } from '../../../utils/TestUtils';
+import { mount, ReactWrapper } from 'enzyme';
+import * as React from 'react';
+import { FlatSelect, IFlatSelectProps } from '../FlatSelect';
+import { IReactVaporState } from '../../../ReactVapor';
+import { Provider, Store } from 'react-redux';
+import { FlatSelectConnected } from '../FlatSelectConnected';
+import { IFlatSelectOptionProps } from '../FlatSelectOption';
+import { clearState } from '../../../utils/ReduxUtils';
+import { selectFlatSelect } from '../FlatSelectActions';
+
+describe('FlatSelect', () => {
+
+  describe('<FlatSelectConnected />', () => {
+    let wrapper: ReactWrapper<any, any>;
+    let flatSelect: ReactWrapper<IFlatSelectProps, void>;
+    let store: Store<IReactVaporState>;
+
+    let id: string = 'flatSelect';
+    let defaultOptions: IFlatSelectOptionProps[] = [{
+      option: { content: 'test' },
+    }, {
+      option: { content: 'test 1' },
```

Comment:
Have you tried putting JSX or a class instead of a string in content? I'm not sure everything will go as expected

Reply:
![image](https://user-images.githubusercontent.com/7167202/33151313-97dc53c0-cfa5-11e7-9ba8-b62208464d98.png)
Work!